### DISCUSSION TITLE
Changing settings of Copilot created pipeline buckets

### DISCUSSION THREAD

**Discussion by gidsg** [aws/copilot-cli#5394] [Q&A] [answered]
We received feedback from a penetration test of our Copilot environment that some of the settings on the copilot created resources need to be changed. 

Note: we are not using the `copilot pipeline` feature, however, it appears Copilot still creates some S3 buckets as part of the environment e.g.
`stackset-$APP-infra-pipelinebuiltartifactbuc-$RANDOM_STRING`

For example we were asked: 

> Add the condition below to all the S3 buckets:

```"Condition":
{
"Bool":
{
"aws:SecureTransport": "false"
}
}
```
Is it possible to adjust the settings Copilot uses when creating these buckets, so this is not a 'clickops' task which needs to be done after app creation?

**Comment by KollaAdithya** [comment, marked as answer]
Hello @gidsg !
Seems like a similar request #5389 and I have a open PR to address this issue #5393 
As a workaround I think you need to manually change the bucket policy.
